WordPress/src/main/java/org/wordpress/android/ui/reader/subfilter/SubfilterListItem.kt

```diff
@@ -68,7 +69,7 @@ sealed class SubfilterListItem(val type: ItemType, val isTrackedItem: Boolean =
         override val label: UiString = if (blog.name.isNotEmpty()) {
             UiStringText(blog.name)
         } else {
-            UiStringRes(R.string.reader_untitled_post)
+            UiStringText(if (blog.url.isNotEmpty()) UrlUtils.getHost(blog.url) else "")
```

Comment:
❓ Should we continue to have `R.string.reader_untitled_post` as a fallback when blog URL is also is empty instead of `""`?